misc/src/main/java/com/ihren/misc/mapper/ObjectMapperFactory.java

```diff
@@ -0,0 +1,11 @@
+package com.ihren.misc.mapper;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class ObjectMapperFactory {
```

Comment:
This class is not necessary. You can call directly constructor in appropriate places